CHANGES.md

```diff
@@ -7,9 +7,20 @@ Version 0.1.1
 To be released.
 
  -  Relicensed under [LGPL 2.1] or later.
+ -  Added more implicit conversions.  [[#1]]
+     -  `Integer` <-> `short`
+     -  `Integer` <-> `ushort`
+     -  `Integer` -> `int`
+     -  `Integer` <-> `uint`
+     -  `Integer` <-> `long`
+     -  `Integer` <-> `ulong`
+     -  `Binary` -> `bytes`
```

Comment:
Unpaired.

Reply:
It was because it isn't added by this pull-request. even so, does this need to be added?

Reply:
Ah, okay. Fair enough.

Reply:
```suggestion
     -  `Binary` → `bytes`
```